### PR TITLE
migrate queue/podgroup to v1alpha2

### PR DESCRIPTION
**Pull request by hzxuzhonghu**
fixes: #357

### CODE REVIEW

**Comment by hzxuzhonghu** [comment]
I will rebase if everything is ok.

File: pkg/admission/admit_job.go [hunk 1]
```diff
@@ -151,8 +151,11 @@ func validateJob(job v1alpha1.Job, reviewResponse *v1beta1.AdmissionResponse) st
 	}
 
 	// Check whether Queue already present or not
+	// TODO: deprecate v1alpha1
 	if _, err := KubeBatchClientSet.SchedulingV1alpha1().Queues().Get(job.Spec.Queue, metav1.GetOptions{}); err != nil {
```
Comment (TommyLike) [inline pkg/admission/admit_job.go]: Try v2 first.

**Comment by TommyLike** [comment]
looks good to me, please rebase it.

File: pkg/admission/admit_job.go (hunk 1 above)
Comment (hzxuzhonghu) [reply pkg/admission/admit_job.go]: ok

**Comment by TommyLike** [comment]
```
test/e2e/admission.go:100:19: context.vkclient undefined (type *context has no field or method vkclient)
test/e2e/admission.go:150:19: context.vkclient undefined (type *context has no field or method vkclient)
```
need another fix.

**Pull request merged**